src/include/mempool.h

```diff
@@ -150,12 +150,16 @@ namespace mempool {
   f(bluestore_alloc)		      \
   f(bluestore_cache_data)	      \
   f(bluestore_cache_onode)	      \
+  f(bluestore_cache_meta)	      \
   f(bluestore_cache_other)	      \
+  f(bluestore_collections)	      \
```

Comment:
there aren't many collections, and the structures are fairly small - pointers or coll_t -  so I'm not sure it's worth adding a separate category - these could stay in bluestore_cache_other